app-admin/prelude-manager/files/prelude-manager-3.1.0-run.patch

```diff
@@ -0,0 +1,13 @@
+When : everytime
+Why : since systemd, run dir has changed
```

Comment:
I am not sure if this information helps much. Let's just change it to a single non-awkward sentence: `Fix the run dir to comply with the Gentoo FS layout.`